README.md

```diff
@@ -192,8 +192,8 @@ scanners - ragel scanner definitions. Because it produces Java,
            See ExampleScanner.java.rl.
 src - source directory
 test - test source directory
-gen-src - generated scanner sources
```

Comment:
"gen-src" is no more valid as we have removed ant build. The generated scanner sources will be found at "src/generated/java"